### PR TITLE
docs(CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by justinmk3**
## Problem

Missing some important resources in CONTRIBUTING.md

## Solution

Update CONTRIBUTING.md

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,9 +74,12 @@ You can also use these NPM tasks (see `npm run` for the full list):
 
 ## Develop
 
-### Code guidelines
+### Guidelines
 
-See [CODE_GUIDELINES.md](./docs/CODE_GUIDELINES.md) for coding conventions.
+- Project practices and patterns: [CODE_GUIDELINES.md](./docs/CODE_GUIDELINES.md)
+- [VS Code UI guide](https://code.visualstudio.com/docs/getstarted/userinterface)
```
Comment (bryceitoc9) [inline CONTRIBUTING.md:80]: I feel like this one could be moved to the CODE_GUIDELINES with some best practices? UI guide doesn't mean much unless we say what we'll do with it.

**Review by bryceitoc9 — APPROVED** [review]
Won't block on my comment but consider it?

File: CONTRIBUTING.md (hunk 1 above)
Comment (JadenSimon) [reply CONTRIBUTING.md:80]: Arguably this should be an entirely different document about UI.

Comment (justinmk3) [reply CONTRIBUTING.md]: Added it there for now.

I am wondering if we will need  our own UI_GUIDELINES.md soon.

**Pull request merged**